scripts/env.d/vars

```diff
@@ -13,3 +13,17 @@ case "$(uname)" in
 esac
 
 # vim: set et nu nobomb fenc=utf8 ft=bash ff=unix sw=2 ts=2:
+
+#----------------------------------------------------------------------------------------------------
+#    make sure system make is installed and reachable 
+#----------------------------------------------------------------------------------------------------
+#  find make
+MAKE=$(which make 2>/dev/null)
+
+if (( $? == 1 )); then 
+  echo " "
```

Comment:
`echo " "` echo an empty space is equivalent to `echo ""`. Do you prefer which one?